plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -21,25 +21,6 @@
          </moduleType>
       </runtimeType>
    </extension>
-   
-   <extension point="org.eclipse.wst.server.core.serverTypes">
-      <serverType
-            hasConfiguration="false"
-            behaviourClass="org.eclipse.jst.server.generic.core.internal.ExternalServerBehaviour"
-            class="org.eclipse.jst.server.generic.core.internal.GenericServer"
-            description="App Engine server that delegates to gcloud (GSF)"
-            id="com.google.cloud.tools.eclipse.gcloud.server"
-            initialState="stopped"
-            launchModes="run,debug"
-            launchConfigId="org.eclipse.jst.server.generic.core.ExternalLaunchConfigurationType"
-            name="%gsfServerTypeName"
-            runtime="true"
-            runtimeTypeId="com.google.cloud.tools.eclipse.gcloud.runtime"
-            startBeforePublish="true"
-            startTimeout="120000"
-            stopTimeout="15000">
-      </serverType>
-   </extension>
 
```

Comment:
The classes are all part of Eclipse so we're not removing them.

Commenting out code you want to get rid of is too wishy washy. If we need it back, that's what git is for.